tests/CMakeLists.txt

```
@@ -1,24 +1,25 @@
 include(Catch)
 
-add_executable(tests)
+set(TEST_TARGET_NAME ${PROJECT_NAME}_tests)
+add_executable(${PROJECT_NAME})
```

Shouldn't it be 
```suggestion
add_executable(${TEST_TARGET_NAME})
```
?

---

Yes, good catch. I don't know how I missed that in my latest typo fix push.